packages/page-calendar/src/Day.tsx

```diff
@@ -50,7 +51,14 @@ function Day ({ className, date, hasNextDay, now, scheduled, setNextDay, setPrev
   return (
     <div className={className}>
       <h1>
-        <div className='highlight--color'>{date.getDate()} {monthRef.current[date.getMonth()]} {date.getFullYear()} {isToday && <DayTime />}</div>
+        <div className='highlight--color'>
+          <Button
+            className='all-events-button'
+            icon={'list'}
+            onClick={() => setView(true)}
```

Comment:
Would prefer this via `useCallback` so it doesn't get re-created on each render. (And then forces the child to re-render as well)